metalift/synthesize_cvc5.py

```diff
@@ -104,10 +104,18 @@ def toExpr(
     expr_uni = {"not": Not}
     if isinstance(ast, list):
         if ast[0] in expr_bi.keys():
-            return expr_bi[ast[0]](
-                toExpr(ast[1], funName, returnType, varType, letVars),
-                toExpr(ast[2], funName, returnType, varType, letVars),
-            )
+            if len(ast) >= 3:
```

Comment:
This should probably be `== 3`, so that we don't discard extra arguments. Similarly below we should probably check that `len(ast) == 2 and ast[0] == "-"` to be extra precise

Reply:
Added checks, fixed length.